src/app/landing/lazyLoading.js

```diff
@@ -0,0 +1,29 @@
+$(document).ready(function(){
```

Comment:
This is also framework mixing. It's using Jquery. We should rather not do this. Let's discuss in detail.